README.MD

```diff
@@ -28,10 +27,19 @@ import { render } from 'react-dom'
 import { createStore, combineReducers } from 'redux';
 
 // 1. Connect your data, similar to react-redux @connect
+// Decorator version
 @asyncConnect({
-  lunch: (params, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
+  lunch: (params, store, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
 })
 class App extends React.Component {
+  // 1. Same as the decorator but as a function
+  static reduxAsyncConnect(params, store, helpers) {
+    const {dispatch, getState} = store;
+    if (!helpers.isLoaded(getState())) {
+      return Promise.all([dispatch(loadWidgets())]);
```

Comment:
there is no loadWidgets defined...

may be it's better here to add just some setTimeout?
this method is just for delay rendering - you can do here whatever you want. dispatch actions in particular